BaseFilter/sections/css_extended.txt

```diff
@@ -2097,6 +2097,7 @@ porntrex.com#?#.content > div[class]:has(> div[style*="text-align"] > center > s
 freshnewsasia.com#?#div[itemprop="articleBody"] > div[style]:has(> a[href*="/banner/"])
 freshnewsasia.com#?#.row-fluid > div#aside:has(a[href*="/banner/"])
 instacart.ca,instacart.com#?#div#store-wrapper div[class] > ul li:has(div[aria-label="Product"], img[alt="Sponsored"])
+instacart.ca,instacart.com#?#div#store-wrapper div.e-ijs5rh:has(span:contains(Sponsor), ul[class^="u-noscrollbar "])
```

Comment:
* Please split into two rules. This is not a mistake, just we decided not to use rules with selector enumeration.

* `ul[class^="u-noscrollbar "]` matches a useful element
https://www.instacart.ca/store/super-c/storefront

<details>
<summary>Details</summary>

![image](https://github.com/AdguardTeam/AdguardFilters/assets/8361299/0db34d28-aab2-4ddf-85c9-e4858ae68a24)


</details>